sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2801,16 +2803,24 @@ SIREPO.app.directive('jRangeSlider', function(appState, panelState) {
 
             $scope.display = (range) => {
                 function toLog(val, r) {
-                    return $scope.formatFloat(SIREPO.UTILS.linearToLog(val, r.min, r.max, r.step));
+                    return formatFloat(SIREPO.UTILS.linearToLog(val, r.min, r.max, r.step));
                 }
 
                 const v = range.val;
                 if (range.space === 'linear') {
-                    return Array.isArray(v) ? v.map(x => $scope.formatFloat(x)) : $scope.formatFloat(v);
+                    return Array.isArray(v) ? v.map(x => formatFloat(x)) : formatFloat(v);
+                }
+                return Array.isArray(v) ? v.map(x => formatFloat(toLog(x, range))) : formatFloat(toLog(v));
+            };
+
+            const formatFloat = val => SIREPO.UTILS.formatFloat(val, 4);
+
+            $scope.formatMinMax = val => {
```

Comment:
We've decided on using parens around single parameters: `const formatFloat = (val) => {...}` etc.